jupyterlab_wipp_plugin_creator/src/sidebar.tsx

```diff
@@ -0,0 +1,160 @@
+import { JupyterFrontEnd } from '@jupyterlab/application';
+import { INotebookTracker } from '@jupyterlab/notebook'
+import { IConsoleTracker } from '@jupyterlab/console';
+import { Widget, PanelLayout } from '@lumino/widgets';
+import { SchemaForm } from '@deathbeds/jupyterlab-rjsf';
+import { ToolbarButton } from '@jupyterlab/apputils';
+import { runIcon } from '@jupyterlab/ui-components';
+import { AddedFileWidget } from './addedFilesWidget'
+import { IStateDB } from '@jupyterlab/statedb'
+import { requestAPI } from './handler';
+
+export class Creator_Sidebar extends Widget {
```

Comment:
Please use `PascalCase` for class names

For reference, see [this](https://basarat.gitbook.io/typescript/styleguide#class)